File: homework_6/calculations/Multiplication.java
package calculations;

//класс для нахождения произведения двух чисел
public class Multiplication implements I__Calculator{
    private double a;
    private double b;

    public Multiplication(double a, double b){
        this.a = a;
        this.b = b;
    }

    @Override
    public double getResult(){
        return a * b;
    }
}

File: homework_7/calculations/multiplication.py
from calculations.I__calculator import Calculations

"""наследуем класс Multiplication от базового Calculations"""
class Multiplication(Calculations):
    def __init__(self, a, b):
        self.a = a
        self.b = b
    
    """переопределяем абстрактый метод"""
    def get_result(self):
        return self.a * self.b